Error building docker image

From https://164938-36831081-gh.circle-artifacts.com/0/artifacts/fxa-builder.log

```
#10 573.0 fxa-auth-server: ../fxa-shared/l10n/localizeTimestamp.ts(5,33): error TS7016: Could not find a declaration file for module 'accept-language'. '/fxa/packages/fxa-shared/node_modules/accept-language/index.js' implicitly has an 'any' type.
#10 573.0 fxa-auth-server:   Try `npm install @types/accept-language` if it exists or add a new declaration (.d.ts) file containing `declare module 'accept-language';`
```

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-1785)
┆Issue Number: FXA-1785

--- a/packages/fxa-auth-server/tsconfig.json
+++ b/packages/fxa-auth-server/tsconfig.json
@@ -14,11 +14,9 @@
     "allowSyntheticDefaultImports": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "types": [
-      "node"
-    ],
     "typeRoots": [
       "./types",
+      "../fxa-shared/types",
       "node_modules/@types"
     ]
   },

